pkgs/tools/networking/hysteria/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "hysteria";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "HyNetwork";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "sha256-V+umf7+qRANSNsMrU1Vij3ni6ayq/d41xSy3o+7sEHQ=";
+  };
+
+  vendorSha256 = "sha256-oxCZ4+E3kffHr8ca9BKCSYcSWQ8jwpzrFs0fvCvZyJE=";
+  proxyVendor = true;
+
+  # Network required
+  doCheck = false;
+
+  meta = with lib; {
+    description = "feature-packed proxy & relay utility optimized for lossy, unstable connections";
```

Comment:
```suggestion
    description = "A feature-packed proxy & relay utility";
```

Reply:
What's the motivation for this change? I know the guidelines say keep it short, but I'd say the previous version is also pretty short...

Reply:
Feels that the description of the second half is superfluous, not because of length. But this suggestion should be my fault.

Reply:
Since the description comes from upstream, and is short enough, I think it's fine.

For example, look at how it renders on https://search.nixos.org (obviously I edited the page to show this in comparison to another package):

<img width="683" alt="image" src="https://user-images.githubusercontent.com/78392041/183341568-d50f1457-e7fd-4d7a-910e-c02834e68197.png">

Reply:
@oluceps, I agree with him. Sorry for my previous suggestion.

Reply:
Modified. thx